guides/common/modules/proc_integrating-idm-dns-with-tsig-authentication.adoc

```diff
@@ -0,0 +1,64 @@
+[id="integrating-idm-dns-with-tsig-authentication"]
+= Integrating IdM DNS with TSIG authentication
+
+You can configure an IdM server to use the secret key transaction authentication for DNS (TSIG) technology that uses the `rndc.key` key file for authentication.
+The TSIG protocol is defined in https://tools.ietf.org/html/rfc2845[RFC2845].
+
+
+.Prerequisites
+
+* You must ensure the IdM server is deployed and the host-based firewall is configured correctly.
```

Comment:
Future improvement: in upstream we could mention FreeIPA instead of IdM. We have `{FreeIPA}` as an attribute that is unbranded upstream and branded downstream.

Reply:
This was easy to implement. Updated in the entire guide.